### DISCUSSION TITLE
Can I generate a static webpage and normally use it without a server?

### DISCUSSION THREAD

**Discussion by bienieck** [foambubble/foam#1247] [Q&A] [answered]
I want to generate a static webpage and run it without a server. I want to zip it and send it to my colleagues and therefore I would like to avoid instructing them on how to set up a local server.

I know that [MkDocs](https://github.com/mkdocs/mkdocs/issues/526) as well as [Material](https://squidfunk.github.io/mkdocs-material/setup/building-for-offline-usage/) docs can generate such "offline" static files. How can I accomplish this with Foam?

**Comment by riccardoferretti** [comment, marked as answer]
Foam doesn't natively provide publishing, only a few building blocks to help integrating with templates/frameworks that can be used to publish one's notes.
I like your idea, at the moment though I don't believe that any of the publishing templates support that. See https://github.com/foambubble/foam/tree/master/docs/user/publishing
